diff --git a/apple/Elements/RNSVGImage.mm b/apple/Elements/RNSVGImage.mm
index 243c24c1b..d5be525e4 100644
--- a/apple/Elements/RNSVGImage.mm
+++ b/apple/Elements/RNSVGImage.mm
@@ -130,8 +130,11 @@ - (void)_setStateAndResubscribeImageResponseObserver:(RNSVGImageShadowNode::Conc
 }

 #pragma mark - RCTImageResponseDelegate
-
+#if !TARGET_OS_OSX // [macOS]
 - (void)didReceiveImage:(UIImage *)image metadata:(id)metadata fromObserver:(void const *)observer
+#else // [macOS
+- (void)didReceiveImage:(NSImage *)image metadata:(id)metadata fromObserver:(void const *)observer
+#endif // macOS]
 {
   if (!_eventEmitter || !_state) {
     // Notifications are delivered asynchronously and might arrive after the view is already recycled.
