From e34f6eaae468d0499b544815f67a73c6f75e9d56 Mon Sep 17 00:00:00 2001
From: William Candillon <wcandillon@gmail.com>
Date: Sat, 18 Oct 2025 18:45:06 +0200
Subject: [PATCH] Remove unavailable platform on simulator

---
 gn/skia/BUILD.gn | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/gn/skia/BUILD.gn b/gn/skia/BUILD.gn
index 8e42db771b..30048a1d43 100644
--- a/gn/skia/BUILD.gn
+++ b/gn/skia/BUILD.gn
@@ -199,9 +199,13 @@ config("default") {
       _arch_flags = [
         "-arch",
         "arm64",
-        "-arch",
-        "arm64e",
       ]
+      if (!ios_use_simulator) {
+        _arch_flags += [
+          "-arch",
+          "arm64e",
+        ]
+      }
     } else if (current_cpu == "x86") {
       _arch_flags = [
         "-arch",
-- 
2.42.0

